server/.env.example

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=
+CLIENT_URL=
+JWT_REFRESH_SECRET=
+JWT_SECRET=
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
```

Comment:
@Aksh-Bansal-dev add a URL to the page where users need to register for google client id and secret.

Reply:
I think it will be better to add a URL to the official docs.

Reply:
@Aksh-Bansal-dev no URL added here.